### PR TITLE
Remove releasing extension from PR yml

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: .github/actions/build-vsix/action.yml [hunk 1]
```diff
@@ -1,11 +1,6 @@
 name: 'Build VSIX'
 description: "Build the extension's VSIX"
 
-inputs:
-  package_command:
-     description: 'Package.json package script'
-     required: false
-     default: 'package'
```
Comment (DonJayamanne) [inline .github/actions/build-vsix/action.yml:8]: We don't need any of these anymore as we now use the azure pipelines for releases.

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -69,68 +59,10 @@ jobs:
...
-        run: |
-          echo "package_command=package" >> $GITHUB_ENV
-
-      - name: pr
-        if: github.event_name == 'pull_request'
-        run: |
-          echo "vsix_name=ms-toolsai-jupyter-insiders.vsix" >> $GITHUB_ENV
-          echo "test_matrix_os=[\"ubuntu-latest\", \"windows-latest\"]" >> $GITHUB_ENV
-
-      - name: pr release push
```
Comment (DonJayamanne) [inline .github/workflows/build-test.yml:95]: We don't need any of these anymore as we now use the azure pipelines for releases.

File: src/test/standardTest.node.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ async function installPythonExtension(vscodeExecutablePath: string) {
     // Pick python extension to use based on environment variable. Insiders can be flakey so
     // have the capability to turn it off/on.
     const pythonVSIX =
-        process.env.VSC_JUPTYER_PYTHON_EXTENSION_VERSION === 'insiders'
+        process.env.VSC_JUPYTER_PYTHON_EXTENSION_VERSION === 'insiders'
```
Comment (sadasant) [inline src/test/standardTest.node.ts:61]: I'm not sure why this is labeled as a change 😅 I can't see the difference 😵

**Review by sadasant — APPROVED** [review]

File: src/test/standardTest.node.ts (hunk 3 above)
Comment (DonJayamanne) [reply src/test/standardTest.node.ts:61]: JUPYTER vs JUPTYER , the `TY` are swapped.
I'm 200% certain I introduced this typo

**Pull request merged**